plugin/storage/es/options.go

```diff
@@ -79,6 +79,36 @@ const (
 	defaultSendGetBodyAs          = ""
 )
 
+var defaultConfig = config.Configuration{
```

Comment:
the way you're using this var is not safe, because can be modified later. It's better to define this as a function tht always returns a new instance.